packages/list/index.tsx

```diff
@@ -185,16 +200,30 @@ export default class List<T extends HTMLElement = HTMLElement> extends React.Com
           listItem.setAttribute(attr, value);
         }
       },
+      /**
+       * Pushes class name to state.listItemClassNames[listItemIndex] if it doesn't yet exist.
+       */
       addClassForElementIndex: (index, className) => {
-        const listItem = this.listElements[index];
-        if (listItem) {
-          listItem.classList.add(className);
+        const {listItemClassNames} = this.state;
```

Comment:
Can this be a set like you did for menu surface?

Reply:
ya it can be a set, but Set is actually not supported on IE11. I created this bug https://github.com/material-components/material-components-web-react/issues/700 to fix that. About 70 - 80% of our components (maybe all use Set), so I'm not sure if it matters here. What do you think?